packages/grid/src/vaadin-grid-selection-column-base-mixin.d.ts

```diff
@@ -35,6 +35,12 @@ export declare class GridSelectionColumnBaseMixinClass<TItem> {
    */
   autoSelect: boolean;
 
+  /**
+   * When true, rows can be selected by dragging mouse cursor over selection column.
```

Comment:
nit: Could be updated not to mention the mouse cursor specifically since the feature also works on touch.

Reply:
Done